Sources/SwiftDocC/Utility/LogHandle.swift

```diff
@@ -51,7 +54,7 @@ public enum LogHandle: TextOutputStream {
         case .file(let fileHandle):
             fileHandle.write(Data(string.utf8))
         case .memory(let storage):
-            storage.text.append(string)
+            storage._text.sync { $0.append(string) }
```

Comment:
Do we need to synchronize `fileHandle` in the same way? Or does macOS/Linux guarantee that writing to a file is thread safe - i.e. synchronized at a lower level? 

Same question for stdout and stderr...

Reply:
I haven't ever encountered any issues with writing to a file and I've never seen a pattern of synchronizing around `fileHandle.write` or `print` but I can't say for sure.